hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -445,6 +453,24 @@ public Blob(long fromVersion, long toVersion) {
          */
         public abstract InputStream getInputStream() throws IOException;
 
+        /**
+         * Blobs can be of types {@code SNAPSHOT}, {@code DELTA} or {@code REVERSE_DELTA}.
+         */
+        public enum BlobType {
+            SNAPSHOT("snapshot"),
```

Comment:
Do you need a `String` field whose value is the lower case of the enum value?

Reply:
Discussed offline, leaving as-is for consistency.